Sources/Services/Health/Health.swift

```diff
@@ -16,62 +16,71 @@
 
 import GRPCCore
 
-/// A coupled Health service and provider.
+/// ``Health`` is gRPC’s mechanism for checking whether a server is able to handle RPCs. Its semantics are documented in
+/// https://github.com/grpc/grpc/blob/master/doc/health-checking.md.
+///
```

Comment:
We should go into more detail here, as an example:


```
/// Each ``Health`` has two objects, a ``Health/Service-swift.struct`` and a
/// ``Health/Provider-swift.struct``. The `service` implements the `Health` service from the
/// `grpc.health.v1` package. It can be registered with your server like any other service.
///
/// The ``Health/Service-swift.struct`` service doesn't know about other services running on the
/// server so you must provide it with information about the status of services via
/// the ``Health/Provider-swift.struct``.
///
/// To make this easier, generated code for services includes an extension to `ServiceDescriptor`
/// so you can easily specify which service is being updated. The following shows an example
/// of creating a health service and updating the state of the 'Foo' service in the 'bar' package.
///
/// ```
/// let health = Health()
/// let server = GRPCServer(transport: transport, services: [health, FooService()])
/// health.provider.updateStatus(.service, forService: .bar_Foo)
/// ```
```